components/demo/demo-page.js

```diff
@@ -20,7 +20,8 @@ class DemoPage extends LitElement {
 
 	static get properties() {
 		return {
-			pageTitle: { type: String, attribute: 'page-title' }
+			pageTitle: { type: String, attribute: 'page-title' },
+			noScroll: { type: Boolean, reflect: true, attribute: 'no-scroll' }
```

Comment:
Should `noScroll` be private?

Reply:
Changed this to an internal state and class instead.